build/configs/cy4390x/cy4390x_demo/Make.defs

```diff
@@ -0,0 +1,148 @@
+############################################################################
+#
+# Copyright 2017, Cypress Semiconductor Corporation or a subsidiary of 
```

Comment:
2017 => 2018  in all  files which includes the licensing info

Reply:
it will be changed as your guidance.

BR, HJ Lee.